lib/dry/validation/result.rb

```diff
@@ -81,3 +81,11 @@ def result_ast
     end
   end
 end
+
+begin
```

Comment:
Yes. I merged it in

On 10 September 2016 at 20:16:38, Nikita Shilnikov (notifications@github.com)
wrote:

> In lib/dry/validation/result.rb
> <https://github.com/dry-rb/dry-validation/pull/238#discussion_r78281513>:
>
> > @@ -81,3 +81,11 @@ def result_ast
> >      end
> >    end
> >  end
> > +
> > +begin
>
> @solnic <https://github.com/solnic> would it be enough dry-rb/dry-core#3
> <https://github.com/dry-rb/dry-core/pull/3>?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/dry-rb/dry-validation/pull/238/files/589c33347bd0490ac3258b751bc3e40ed8970fe4#r78281513>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAAEKvmGuHDwld78iU57haGN5zzjFpBOks5qovQGgaJpZM4JnxG5>
> .
>